src/core.jl

```diff
@@ -141,6 +141,7 @@ function nrows(X)
     return length(cols[1])
 end
 nrows(y::AbstractVector) = length(y)
+nrows(X::AbstractMatrix) = size(X, 1)
 
```

Comment:
Oh, I see. We're not using `nrows` from MLJBase.jl - I forgot. Thanks for humouring me with the explanation!